OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -130,23 +130,23 @@ public void GetByProviderId_WhenIdIsInvalid_ShouldThrowArgumentOutOfRangeExcepti
         public async Task GetByProviderId_WhenThereAreWorkshops_ShouldReturnOkResultObject(long id)
         {
             // Arrange
-            workshopServiceMoq.Setup(x => x.GetByProviderId(id)).ReturnsAsync(workshops.Where(x => x.ProviderId == id));
+            workshopServiceMoq.Setup(x => x.GetByProviderId(id)).ReturnsAsync(workshopCards.Where(x => x.ProviderId == id));
 
             // Act
             var result = await controller.GetByProviderId(id).ConfigureAwait(false) as OkObjectResult;
 
             // Assert
             Assert.That(result, Is.Not.Null);
             Assert.AreEqual(Ok, result.StatusCode);
-            Assert.AreEqual(2, (result.Value as IEnumerable<WorkshopDTO>).Count());
+            Assert.AreEqual(2, (result.Value as IEnumerable<WorkshopCard>).Count());
         }
 
         [Test]
         [TestCase(3)]
```

Comment:
Why 3 ?
I think it will be better to configure mock with nonempty collection and get id value from outside of the existing ids. like
`workshops.Select(w => w.Id).Max() + 1`  
`workshops.Select(w => w.Id).Max() + randomNumber`
You can randomize your data with [RandomAttribute](https://docs.nunit.org/articles/nunit/writing-tests/attributes/random.html)